.buildkite/scripts/steps/build_and_test.sh

```diff
@@ -32,12 +34,6 @@ if [[ -n "$BUILDKITE_PULL_REQUEST" && -n "$VERSION_QUALIFIER" ]] ; then
     exit 2
 fi
 
-# Tests must be run in PR builds
-if [[ -n "$BUILDKITE_PULL_REQUEST" && "$RUN_TESTS" = false ]] ; then
-    echo "RUN_TESTS should not be false PR builds"
-    exit 3
-fi
-
```

Comment:
Please can you add in this functionality: https://github.com/elastic/ml-cpp/blob/5b3696ade2f54e4c3c7164d9e252df55111a9c72/dev-tools/jenkins_cross_compile.sh#L90-L96

In other words, in PR builds always do a debug build when cross compiling for linux-aarch64. The reasoning is in the code comment.